README.md

```diff
@@ -37,50 +31,61 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 
 
 <!-- This heading should always match the name of the root level module (aka the repo name) -->
-## terraform-ibm-module-template
+## terraform-ibm-trusted-profile
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
+```hcl
+module "trusted_profile {
+  source                      = "terraform-ibm-modules/trusted-profile/ibm"
+  version                     = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
+  trusted_profile_name        = "example-profile"
+  trusted_profile_description = "Example Trusted Profile"
+
+  trusted_profile_policies = [{
+    roles = ["Reader", "Viewer"]
+    resources = [{
+      service           = "kms"
+    }]
+  }]
+
+  trusted_profile_claim_rules = [{
+    conditions = [{
+      claim    = "Group"
+      operator = "CONTAINS"
+      value    = "\"Admin\""
+    }]
+
+    type    = "Profile-CR"
+    cr_type = "VSI"
+  }]
+
+  trusted_profile_links = [{
+    cr_type = "VSI"
+    links = [{
+      crn = ibm_is_instance.vsi.crn # Existing Infrastructure CRN
+    }]
+  }]
+}
+```
 
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
+#### Using the variables
 
-```hcl
+The 3 variables `trusted_profile_policies`, `trusted_profile_claim_rules`, and `trusted_profile_links` are lists of objects whose fields are mapped out to match the arguments for the provider, for more information on the variables visit the following provider documentation:
 
-```
+* [trusted_profile_policies](https://registry.terraform.io/providers/IBM-Cloud/ibm/1.53.0/docs/resources/iam_trusted_profile_policy)
+* [trusted_profile_claim_rules](https://registry.terraform.io/providers/IBM-Cloud/ibm/1.53.0/docs/resources/iam_trusted_profile_claim_rule)
+* [trusted_profile_links](https://registry.terraform.io/providers/IBM-Cloud/ibm/1.53.0/docs/resources/iam_trusted_profile_link)
```

Comment:
this is locked into `1.53.0` provider version docs. If you replace that with the string "latest" it will always bring you to latest docs.